src/app/qgsstatusbarcoordinateswidget.cpp

```diff
@@ -117,7 +119,19 @@ void QgsStatusBarCoordinatesWidget::validateCoordinates()
   {
     return;
   }
-  if ( mLineEdit->text() == QLatin1String( "dizzy" ) )
+  else if ( mLineEdit->text() == QLatin1String( "world" ) )
+  {
+    this->world();
```

Comment:
Remove this-> prefix (elsewhere too)

Reply:
Hah my python alter ego feels there should be 'this' or 'self' behind every call :-P ... will do...